src/Dfe.PlanTech.Web/Dfe.PlanTech.Web.csproj

```diff
@@ -34,12 +34,17 @@
         <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.4" />
         <PackageReference Include="Microsoft.Extensions.Http" Version="8.0.0" />
         <PackageReference Include="Microsoft.Extensions.Http.Polly" Version="8.0.4" />
+        <PackageReference Include="System.Linq.Async" Version="6.0.1" />
     </ItemGroup>
 
     <ItemGroup>
         <None Remove="Views\Recommendations\" />
     </ItemGroup>
 
+    <ItemGroup>
```

Comment:
Is this needed? I'm assuming your IDE added it.

Reply:
Ah yeah - probably not, I gather thats for files that exist but you want to exclude
Which makes me think the one on line 40 could surely go as well? Why would that specifically be removed but not any other view folder?

Reply:
Yeah it should be able to go as well. In cases like this I tend to just remove it and see if it still builds and runs :D